cmake/FindBLAS.cmake

```diff
@@ -569,6 +569,15 @@ if(BLA_VENDOR MATCHES "Arm" OR BLA_VENDOR STREQUAL "All")
      set(BLAS_armpl_LIB "${BLAS_armpl_LIB}_mp")
    endif()
 
+   # Arm Performance Libraries contains Fortran code and must be linked against libgfortran for a GCC build.
```

Comment:
QQ, how does this work for LLVM's clang?

Reply:
There's no clang support for the Free version of ArmPL. With the subscription product, it's shipped as apart of the toolchain with companion C (armclang) & Fortran (armflang) compilers, and linked to `libarmflang`, so there's no requirement to add the Fortran lib to the link line at this point.

Reply:
I see. Thanks!